### PR TITLE
PlatformIndicators: make size same as other memberlist icons

### PR DESCRIPTION
**Pull request by mctaylors**
In this PR, I resized platform indicators to 17px (14px for mobile indicator) to be more size consistent with other icons.

Previews:

<details><summary>Default size</summary>

Message preview:
image
List preview:
image
List preview w/ server booster icon:
image (100%)
image (200%)

</details>

<details><summary>Resized</summary>

Message preview:
image
List preview:
image
List preview w/ server booster icon:
image (100%)
image (200%)

</details>

### CODE REVIEW

**Comment by Vendicated** [comment]
they look too small with this change. the previous one was exactly fitted to discord's dimensions
grafik



grafik

grafik

**Comment by mctaylors** [comment]
Oh, I forgot about badge icons, I'll see what I can do.

File: src/plugins/platformIndicators/index.tsx [hunk 1]
```diff
@@ -50,23 +50,26 @@ function Icon(path: string, opts?: { viewBox?: string; width?: number; height?:
...
-const PlatformIcon = ({ platform, status }: { platform: Platform, status: string; }) => {
+const PlatformIcon = ({ platform, status, large }: { platform: Platform, status: string; large?: boolean; }) => {
     const tooltip = platform[0].toUpperCase() + platform.slice(1);
     const Icon = Icons[platform] ?? Icons.desktop;
+    let addSize = 0;
 
-    return <Icon color={`var(--${getStatusColor(status)}`} tooltip={tooltip} />;
+    if (large == true) addSize = 3;
+
+    return <Icon color={`var(--${getStatusColor(status)}`} tooltip={tooltip} addSize={addSize} />;
```
Comment (Vendicated) [inline src/plugins/platformIndicators/index.tsx:67]:
```suggestion
    return <Icon color={`var(--${getStatusColor(status)}`} tooltip={tooltip} addSize={!large ? 0 : 3} />;
```

File: src/plugins/platformIndicators/index.tsx [hunk 2]
```diff
@@ -30,13 +30,13 @@ import { User } from "discord-types/general";
 const SessionsStore = findStoreLazy("SessionsStore");
 
 function Icon(path: string, opts?: { viewBox?: string; width?: number; height?: number; }) {
-    return ({ color, tooltip }: { color: string; tooltip: string; }) => (
+    return ({ color, tooltip, addSize }: { color: string; tooltip: string; addSize: number; }) => (
```
Comment (Vendicated) [inline src/plugins/platformIndicators/index.tsx:33]: weird prop name. you could just name it large like below and make it a boolean, that way the consumer doesn't even have to know about the pixels to add

**Comment by Vendicated** [comment]
i think they're too small in chat now? 
grafik

**Comment by Vendicated** [comment]
thanks!

**Pull request merged**